.github/scripts/check_plugin.sh

```diff
@@ -20,10 +20,10 @@ ODFE_VERSION=$2
 OLDIFS=$IFS
 
 # This script allows users to manually assign parameters
-if [ "$#" -gt 2 ] || [ "$1" = "--help" ] || [ "$1" = "-h" ]
+if [ "$#" -eq 0 ] || [ "$#" -gt 2 ] || [ "$1" = "--help" ] || [ "$1" = "-h" ]
 then
-  echo "Please assign at most 2 parameters when running this script"
-  echo "Example: $0 [\$PLUGIN_TYPES] [\$ODFE_VERSION]"
+  echo "Please assign at least 1 / at most 2 parameters when running this script"
+  echo "Example: $0 \$PLUGIN_TYPES [\$ODFE_VERSION]"
   echo "Example: $0 \"tar\""
   echo "Example: $0 \"rpm,kibana\" \"1.7.0\""
```

Comment:
It would be useful to document what the valid plugin types are and include those in the help message.

Reply:
It is right inside this script so I didnt document it.